packages/docs/components/molecules/AnchorNav/index.md

```diff
@@ -9,3 +9,59 @@
 
 - [Examples](./examples)
 - [JS API](./js-api)
+
+## Usage
+
+This component can be directly imported and defined as a dependency of your application:
+
+```js{2,8}
+import { Base, createApp } from '@studiometa/js-toolkit';
+import { AnchorNav as AnchorNavCore, AnchorNavLink, AnchorNavTarget } from '@studiometa/ui';
```

Comment:
**suggestion:** Remove the import for `AnchorNavLink` and `AnchorNavTarget`

Both components are already registered as child component of the `AnchorNav` component, we don't need to declare them again. This will simplify the usage example.